src/data/FeaturedProjectsInfo.json

```diff
@@ -0,0 +1,39 @@
+[
+    {
+      "title": "Popular Movies",
+      "title-description": "Popular Movies App",
+      "project-description": "A responsive web site displaying popular movies and their details",
+      "repo-name": "project-movies",
+      "netlify-name": "project-movies-neah-viktor",
+      "tags": ["React", "API", "Routes", "Pair-programming"],
+      "image": "/Assets/thumbnail-movies.png"
+    },
+    {
+      "title": "New Music",
+      "title-description": "New music releases App",
+      "project-description": "Pair project that displays recently released music",
+      "repo-name": "project-music-releases",
+      "netlify-name": "neah-and-linnea-music-releases",
+      "tags": ["React", "API", "Pair-programming"],
+      "image": "/Assets/thumbnail-music-releases.png"
```

Comment:
This link doesn't seem to exist:
https://github.com/Archkrull/project-music-releases